.github/CODEOWNERS

```diff
@@ -5,7 +5,18 @@
 # 3. PRs touching any code with a codeowner must be signed off by at least one
 #    person on the code owner team.
 
+# tsc & commcomm
+
 /.github/CODEOWNERS @nodejs/tsc
+/CODE_OF_CONDUCT.md @nodejs/tsc @nodejs/community-committee
+/CONTRIBUTING.md @nodejs/tsc @nodejs/community-committee
+/LICENSE @nodejs/tsc @nodejs/community-committee
+/GOVERNANCE.md @nodejs/tsc @nodejs/community-committee
```

Comment:
So, historically the CommComm has not been an owner of Governance / Contributing of nodejs/node. I'm not saying that we can't make this change (I've already approved this PR) but I do this is a fairly large change from what we've done until now... and AFAICT from the charter of the commcomm it is slightly out of scope imho.

I could see how Code of Conduct, License, and potentially Contributing could fall under ComCoom but I'm a little less easy about including governance, specifically because this is something the TSC is chartered to manage. CommComm members do not automatically get commit rights for this repo, so it seems a bit out of scope. For the record I would say the same thing about auto adding the TSC to the governance document of any working group.

Reply:
I have no objection to @MylesBorins’s comments here.

Reply:
That makes sense, I'll update :)

Reply:
Updated, kept CommComm on License, Contributing and both Code of Conduct files (yes, we have two)

Reply:
@MylesBorins ... while that is true, keep in mind that this doesn't change the process for modifying those. It still requires no objections from TSC to modify any part of the governance. There's likely no harm in CommComm being notified of these changes.

Reply:
I guess this is semantics about if CodeOwners is being used to automate notifications or if it signals ownership / signoff. I was assuming the latter, but my impression seems to be off.

Reply:
It's ... both? ;-) CODEOWNERS is: these are the people who really need to be aware of these changes. Given that we have a Must Object approach rather than a Must Agree approach, however, with the governance docs it still comes down to TSC objections being most critical. That said, if someone from commcomm raises an objection (or anyone else for that matter), there's going to be an effort made to resolve that before moving forward, regardless of CODEOWNERS

Reply:
> That said, if someone from commcomm raises an objection (or anyone else for that matter), there's going to be an effort made to resolve that before moving forward, regardless of CODEOWNERS

Thinking about this a bit more I think what is most important here is that the TSC is onboard with this specific commitment, and since it at least to me appears to be a deviation from current norms that we have broadly made folks aware of this commitment.

What I would not like to see happen is there be an objection in the future that is simply dismissed, or some form of contention that is created by some folks maybe not being ok with this commimtent.

To be clear I'm +1 on making this committment.

Reply:
I wouldn't expect any objections to be dismissed and would be unpleasantly surprised if it happened either today or after we land this. Collaborators can object to **any** changes to this repository that target the default branch, in which case: 1) consensus attempt is made via discussions in the PR; 2) objection is escalated to the TSC, who can try to mediate or go to a vote if mediation fails. We don't have exceptions for specific files *in the current* process.

Either way, CODEOWNERS is not mentioned anywhere in our collaboration docs today, which means it has no clear definition yet, and therefore is only used/considered for notifications.

(just describing what we have today, doesn't mean we can't or should make changes in the future)